lib/Dialect/LLHD/Transforms/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_mlir_dialect_library(MLIRLLHDTransforms
+  ProcessLoweringPass.cpp
+  FunctionEliminationPass.cpp
+
+  DEPENDS
+  MLIRLLHDIncGen
```

Comment:
MLIRLLHDIncGen is unnecessary, since there should be PUBLIC dependency on MLIRLLHD